app/api/biz/handler/job/job_handler.go

```diff
@@ -53,6 +55,25 @@ func SubmitJob(ctx context.Context, c *app.RequestContext) {
 	req.AccessToken = formReq.AccessToken
 	req.Creator = formReq.Creator
 	req.Envs = formReq.Envs
+	req.CPUCount = formReq.CPUCount
+	req.DiskSize = formReq.DiskSize
+	if err := req.IsValid(); err != nil {
+		hlog.Errorf("[Job Handler]Invalid formReq: %v", err)
+		utils.ReturnsJSONError(c, err)
+		return
+	}
+
+	cpuCounts := []int64{2, 4, 8, 16, 32, 48, 64, 80, 96, 128, 224}
```

Comment:
nit: I think this is specific to Google Confidential Space
please add comment here indicating this is values that works for GCP confidential space, but later could be changed.